### PR TITLE
Feature#11 frontend dashboard admin

### LINKED ISSUE #11: Frontend dashboard admin
# REQUIREMENTS

Dashboard view
- Design for the dashboard admin page

# DESIGN
Admin Dashboard

### PR DESCRIPTION
**Pull request by MauricioNC**
# Description

 - New controller for the dashboard_admin view.
 - Deleted the 'wdm' gem from the Gemfile.
 - Added styles using sass for the dashboard_admin desgin.
 
Note:
The 'wdm' gem was deleted because all of we are using a Linux environment

  ## Screenshots
Mockup
image

View
image

### CODE REVIEW

**Comment by DerekAyala** [comment]
@kevinnio @YilKanda could you give us a code review ?

File: app/assets/stylesheets/dashboard_admin.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the dashboard_admin controller here.
```
Comment (xFerchoVJ) [inline app/assets/stylesheets/dashboard_admin.scss]: If you don't need this file, you can delete it

Comment (MauricioNC) [reply app/assets/stylesheets/dashboard_admin.scss]: ok, on it

**Comment by AngelBFdev** [comment]
@ccarruitero can you give us a code review?

File: app/assets/stylesheets/dashboard_admin.scss (hunk 1 above)
Comment (MauricioNC) [reply app/assets/stylesheets/dashboard_admin.scss]: Done.

File: app/views/dashboard_admin/dashboard_admin.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dashboard admin</title>
+</head>
+<body>
```
Comment (YilKanda) [inline app/views/dashboard_admin/dashboard_admin.html.erb]: Why did you leave this code here? Remember that this part is already in the application.erb

File: app/views/dashboard_admin/dashboard_admin.html.erb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      </div>
+      <div class="reports-container">
+        <h1>Reports</h1>
+        <input type="button" value="Attendance by day">
+        <input type="button" value="Average time checked in/out by month">  
+        <input type="button" value="Absence report by month">
+      </div>
+    </div>
+</body>
+</html>
```
Comment (YilKanda) [inline app/views/dashboard_admin/dashboard_admin.html.erb]: Why did you leave this code here? Remember that this part is already in the application.erb

File: test/controllers/dashboard_admin_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class DashboardAdminControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (YilKanda) [inline test/controllers/dashboard_admin_controller_test.rb]: Code that you do not use, delete it.

File: app/helpers/dashboard_admin_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module DashboardAdminHelper
+end
```
Comment (YilKanda) [inline app/helpers/dashboard_admin_helper.rb]: If you don't use it, remove it.

File: app/assets/stylesheets/dashboard_admin.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YilKanda) [inline app/assets/stylesheets/dashboard_admin.scss]: If you don't use it, remove it.

Comment (MauricioNC) [reply app/assets/stylesheets/dashboard_admin.scss]: Done.

File: test/controllers/dashboard_admin_controller_test.rb (hunk 4 above)
Comment (MauricioNC) [reply test/controllers/dashboard_admin_controller_test.rb]: Just delete the code?, I mean, can leave the file in blank? @YilKanda

Comment (YilKanda) [reply test/controllers/dashboard_admin_controller_test.rb]: I am not referring to that, what I am saying is that the code that is not necessary and that for this reason is _commented_, it is better to eliminate it.

In GitHub only your functional code should appear, for this reason, the commented code is not necessary.

Comment (MauricioNC) [reply test/controllers/dashboard_admin_controller_test.rb]: Ok, got it

File: app/views/dashboard_admin/dashboard_admin.html.erb (hunk 2 above)
Comment (MauricioNC) [reply app/views/dashboard_admin/dashboard_admin.html.erb]: Done.

File: app/views/dashboard_admin/dashboard_admin.html.erb (hunk 3 above)
Comment (MauricioNC) [reply app/views/dashboard_admin/dashboard_admin.html.erb]: Done.

File: test/controllers/dashboard_admin_controller_test.rb (hunk 4 above)
Comment (MauricioNC) [reply test/controllers/dashboard_admin_controller_test.rb]: Done.

File: app/helpers/dashboard_admin_helper.rb (hunk 5 above)
Comment (MauricioNC) [reply app/helpers/dashboard_admin_helper.rb]: Done.

**Comment by DerekAyala** [comment]
@kevinnio @YilKanda @xFerchoVJ could you give us one more code review ?

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by xFerchoVJ — APPROVED** [review]
LGTM!

**Pull request merged**